modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/result/AbstractReturnDelegate.java

```diff
@@ -255,7 +263,10 @@ private Object streamingContent(Object value,
       ConnectionHandler connectionHandler = (ConnectionHandler) operationContext.getVariable(CONNECTION_PARAM);
       if (connectionHandler != null && supportsStreaming(operationContext.getComponentModel())) {
         value = componentDecoratorFactory
-            .decorateOutput(new ConnectedInputStreamWrapper((InputStream) value, connectionHandler), correlationId);
+            .decorateOutput(new ConnectedInputStreamWrapper((InputStream) value, connectionHandler,
```

Comment:
what's changed here? if it's only formatting then revert

Reply:
there is change